server/routes/induction/update/qualificationDetailsUpdateController.test.ts

```diff
@@ -51,6 +51,13 @@ describe('qualificationDetailsUpdateController', () => {
       req.session.inductionDto = inductionDto
       const qualificationLevelForm = { qualificationLevel: QualificationLevelValue.LEVEL_3 }
       req.session.qualificationLevelForm = qualificationLevelForm
+      req.session.pageFlowQueue = {
+        pageUrls: [
+          `/prisoners/${prisonNumber}/induction/qualifications`,
+          `/prisoners/${prisonNumber}/induction/qualification-level`,
+        ],
+        currentPageIndex: 2,
```

Comment:
Minor, but the `currentPageIndex` is zero indexed, so this should be `1`

Reply:
It is indeed 0 indexed, but there are a couple of things to consider:

1. `getPreviousPage()` subtracts 1 from the index: `return pageFlowQueue.pageUrls[pageFlowQueue.currentPageIndex - 1]`. If I change this number to 1 then the test breaks.
2. Normally the `routerRequestHandler` would increment the page index, but that's not involved in this unit test.

Reply:
I've added an additional URI, since we agreed late yesterday that the full flow (including `/qualifications-details` which is currently the last page in this flow) should be included.